code/modules/mob/living/carbon/xenomorph/abilities.dm

```diff
@@ -781,6 +783,7 @@
 	mechanics_text = "Spit neurotoxin or acid at your target up to 7 tiles away."
 	ability_name = "xeno spit"
 	keybind_signal = COMSIG_XENOABILITY_XENO_SPIT
+	use_state_flags = XACT_USE_LYING
```

Comment:
No

Reply:
This doesn't allow to spit. This just allow to change spit, cause the buttons are merged

Reply:
Was my initial comment too long? I did already say that it didn't allow you to spit while resting, just to choose your spit type.